reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+            Path to directory containing microsoft strucutes *.geojsons. Used
+            to identify structures to build setbacks from. Files should be
+            by state
+
+        Returns
+        -------
+        structure_state_map : dict
+            Dictionary mapping state to geojson file path
+        """
+        features_state_map = {}
+        for file in os.listdir(structures_dir):
+            if file.endswith('.geojson'):
+                state = file.split('.')[0]
+                state = cls._split_state_name(state)
+                features_state_map[state] = os.path.join(structures_dir, file)
+
+        return features_state_map
+
+    @staticmethod
+    def _parse_features(structure_fpath, crs):
+        """
+        Load structures from geojson, convert to exclusions coordinate system
+
+        Parameters
+        ----------
+        structure_fpath : str
+            Path to Microsoft .geojson of structures in a given state
+        crs : str
+            Coordinate reference system to convert structures geometries into
+
+        Returns
+        -------
+        structures : geopandas.GeoDataFrame
+            Geometries for structures in geojson, in exclusion coordinate
+            system
+        """
+        structures = gpd.read_file(structure_fpath)
+
+        return structures.to_crs(crs)
+
+    def compute_setbacks(self, structure_fpath, max_workers=None,
+                         layer=None, description=None, replace=False):
+        """
+        Compute structure setbacks for all states either in serial or parallel.
+        Existing setbacks are computed if a wind regulations file was supplied
+        during class initialization, otherwise general setbacks are computed
+
+        Parameters
+        ----------
+        features_dir : str
+            Path to directory containing features files. Used
+            to identify features to build setbacks from. Files should be
+            by state
+        max_workers : int, optional
+            Number of workers to use for setback computation, if 1 run in
+            serial, if > 1 run in parallel with that many workers, if None
+            run in parallel on all available cores, by default None
+        layer : str, optional
+            Name of new layer to write to exclusions .h5 file containing
+            computed setbacks, if None do not write to disc, by default None
+        description : str, optional
+            Description of exclusion layer (set as an attribute),
+            by default None
+        replace : bool, optional
+            Flag to replace existing layer data with arr if layer already
+            exists in the exlcusion .h5 file, by default False
+
+        Returns
+        -------
+        setbacks : ndarray
+            Raster array of setbacks, ready to be written to the exclusions
+            .h5 file as a new exclusion layer
+        """
+        setbacks = super().compute_setbacks(structure_fpath, groupby='State',
```

Comment:
I'm of the *opinion* that subclass methods that only call the inherited method via super() with a kwarg change should not be subclass methods. It's just not worth the extra code due to additional complexity and maintenance. Why not remove this subclass method and call `BaseWindSetbacks.compute_setbacks()` with the kwarg in the `subclass.run()` methods?